### PR TITLE
Adding EmialController and helper. Registering emailHelper

### PR DESCRIPTION
**Pull request by 2xFrost**
Adding EmialController and helper. Registering emailHelper. Sending emails works (it work for me)

### CODE REVIEW

File: Data/Repositories/UserRepository.cs [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
 
-        public void GetUserById(int id) { }
+        public UserRepository(DataContext context)
+        {
+            _dataContext = context;
+        }
+
+        public User GetUserById(int id) 
+        {
+            User user = new();
```
Comment (tkarelus) [inline Data/Repositories/UserRepository.cs]: This line is useless

File: Data/Repositories/UserRepository.cs [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+            user = _dataContext.Users.SingleOrDefault(c => c.Id == id);
+
+            if(user != null) return user;
+
+            return null;
```
Comment (tkarelus) [inline Data/Repositories/UserRepository.cs]: You return user if it's not null. And you return null if user is null. So you can just return user without checking anything

File: Services/Services/UserService.cs [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
 namespace Services.Services
 {
-    internal class UserService
+    public class UserService
     {
+
+        private IUserRepository _userRepository;
+
+        public UserService(IUserRepository userRepository) { _userRepository = userRepository; }
+        public string GetEmailOfUser(int id)
```
Comment (tkarelus) [inline Services/Services/UserService.cs:19]: Do you need this method?

Comment (2xFrost) [reply Services/Services/UserService.cs:19]: This method will return email and i will be injecting it to Controller to get emails form db.

**Pull request merged**